tests/integration/s3/test_s3_notifications_lambda.py

```diff
@@ -1,10 +1,11 @@
+import json
 import os
-import time
 
-from localstack.services.awslambda.lambda_utils import LAMBDA_RUNTIME_PYTHON36
-from localstack.utils import testutil
-from localstack.utils.aws import aws_stack
-from localstack.utils.common import retry, short_uid
+import pytest
+
+from localstack.utils.common import retry
+from localstack.utils.common import safe_requests as requests
```

Comment:
Please do not import directly from .common, but from the respective modules in localstack.utils. For retry, this would be localstack.utils.sync for example.
We should change those imports whenever we run across them.